internal/report/report.go

```diff
@@ -101,6 +103,26 @@ func NewPolicyReport(namespace *v1.Namespace) PolicyReport {
 	}
 }
 
+func getClusterReportName(name string) string {
+	return PrefixNameClusterPolicyReport + name
+}
+
+func getNamespacedReportName(namespace string) string {
+	return PrefixNamePolicyReport + namespace
+}
+
+func (r *PolicyReport) CopyResult(result *v1alpha2.PolicyReportResult) {
+	switch result.Result {
+	case StatusFail:
+		r.Summary.Fail++
+	case StatusError:
+		r.Summary.Error++
+	case StatusPass:
+		r.Summary.Pass++
+	}
+	r.Results = append(r.Results, result)
```

Comment:
Shouldn't we be substituting the old result instead of appending a new one?
Maybe I'm misunderstanding the usage of this function in scanner.go.

We have `CopyResult(result)`, and `AddResult(args that create a result)`.
Maybe `CopyResult(result)` should be named `AddResult(result)`, and `AddResult(args to create result)` should be named `CreateResult(args to create result)`.